r18/apply_r17_patch.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+set -ex
+
+for i in erlc shell
+do
+        diff -u ../r17/erl_parse.yrl ../r17/erl_parse_$i.yrl | patch -p2 -o erl_parse_$i.yrl
```

Comment:
As I said -- monkey see, monkey do :D These scripts produce a diff between the stock and the modified `erl_parse.yrl` in the previous release directory and apply the diff to the stock `erl_parse.yrl` in the current directory. Sometimes the merge will be non-trivial (as in the case of R17->R18) but for the most part it works :)